hexyl: add page (#3437)
tldr-pages/tldr: pages/common/hexyl.md

# hexyl

> A simple hex viewer for the terminal. Uses colored output to distinguish different categories of bytes.
> More information: <https://github.com/sharkdp/hexyl>.

- Print the hexadecimal representation of a file:

`hexyl {{file}}`

> - Print the hexadecimal representation of a file, but interpret only n bytes of the input:

Reviewer: Perhaps this will read better?
```suggestion
- Print the hexadecimal representation of the first n bytes of a file:
```

Also, could `file` be `path/to/file` please? Same goes for the other examples.

> `hexyl -n {{bytes_to_read}} {{file}}`

Reviewer 2: Let's change bytes_to_read to n to match with the description.

> - Print bytes N through M of a file:

Reviewer 2: I would suggest to use concrete values for this rather than N and M

`hexyl -r {{N}}:{{M}} {{file}}`

- Print 512 bytes starting at the 1024th byte:

> `hexyl -r 1024:+512 {{file}}`

Reviewer 2: `{{1024}}:+{{512}}`
